imports/components/puzzle.jsx

```diff
@@ -213,8 +213,11 @@ class Puzzle extends React.Component {
       return;
     }
 
-    if (e.altKey && e.key === 'p') {
+    // On MacOS, the altKey modifier has been applied to the key already
+    // before it gets sent to the keyDown, so 'p' is already 'π'.
+    if (e.altKey && (e.key === 'p' || e.key === 'π')) {
       this.delegate.togglePencil();
+      e.stopPropagation();
```

Comment:
I expected this to need `preventDefault` but it needed `stopPropagation` instead.  I didn't go source-diving but presumably the reason is that the actual insertion of characters onto the puzzle is handled by some other event handler, vs being handled by a native input element.